integration-tests/security/src/main/resources/application.properties

```diff
@@ -7,6 +7,11 @@ org.acme.openapi.security.auth.basic_auth/password=test
 quarkus.openapi-generator.codegen.spec.open_weather_yaml.base-package=org.acme.openapi.weather
 quarkus.openapi-generator.open_weather_yaml.auth.app_id.api-key=12345
 
+# #308 use the configured api-key as the Authorization header
+# to avoid case of @HeaderParam("Authorization") in openapi definition creating multiple Authorization headers
```

Comment:
I don't think this comment is needed.
WDYT @ricardozanini?

Reply:
I agree.